static/js/directives/modal.js

```diff
@@ -1,17 +1,38 @@
+/**
+ * Directive for wrapping modal bodies with the modal chrome.
```

Comment:
it's the chrome browser, made out of modals? :brain: :boom: 🇬🇧 
"Directive for boilerplate for modal bodies"